src/components/DocumentationTopic/RelationshipsList.vue

```diff
@@ -128,20 +128,16 @@ export default {
 .relationships-list {
   list-style: none;
 
-  &.column {
-    margin: 0;
```

Comment:
I'm not sure if this was purposeful or accidental, but removing this has the effect of these lists now inheriting the default indentation list style, which slightly indents these lists like so:

<img width="1129" alt="Screen Shot 2022-03-02 at 3 17 29 PM" src="https://user-images.githubusercontent.com/212918/156466009-123073b5-ecde-46bd-8b35-ff7aceea4443.png">